out of order alias type

The following program `sample.go` triggers a panic:

```go
package main

import "fmt"

type MyT T

type T struct {
        Name string
}

func main() {
        fmt.Println(MyT{})
}
```

Expected result:
```console
$ go run ./sample.go
{}
```

Got:
```console
$ yaegi ./sample.go
panic: runtime error: invalid memory address or nil pointer dereference
[signal SIGSEGV: segmentation violation code=0x1 addr=0x0 pc=0x63a0ad]

goroutine 1 [running]:
github.com/containous/yaegi/interp.setExec.func1(0xc0000edf00)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:1617 +0xad
github.com/containous/yaegi/interp.setExec.func1(0xc0000edc00)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:1606 +0x25d
github.com/containous/yaegi/interp.setExec(0xc0000edc00)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:1620 +0xf5
github.com/containous/yaegi/interp.genRun.func1(0xc0000ed700, 0xc00018dd01)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:1413 +0x87
github.com/containous/yaegi/interp.(*node).Walk(0xc0000ed700, 0xc00018dd80, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:112 +0xc9
github.com/containous/yaegi/interp.(*node).Walk(0xc0000ed400, 0xc00018dd80, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:116 +0x70
github.com/containous/yaegi/interp.(*node).Walk(0xc0000ec300, 0xc00018dd80, 0x0)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:116 +0x70
github.com/containous/yaegi/interp.genRun(0xc0000ec300, 0xc0002261e0, 0xc0000c49f8)
        /home/marc/go/src/github.com/containous/yaegi/interp/cfg.go:1405 +0x64
github.com/containous/yaegi/interp.(*Interpreter).Eval(0xc0000eea00, 0xc00015e070, 0x6c, 0x26c, 0xc00015e070, 0x6c, 0x0, 0xc000226180)
        /home/marc/go/src/github.com/containous/yaegi/interp/interp.go:295 +0x35e
main.main()
        /home/marc/go/src/github.com/containous/yaegi/cmd/yaegi/yaegi.go:77 +0x2f9
```

--- a/_test/alias1.go
+++ b/_test/alias1.go
@@ -1,0 +1,16 @@
+package main
+
+import "fmt"
+
+type MyT T
+
+type T struct {
+	Name string
+}
+
+func main() {
+	fmt.Println(MyT{})
+}
+
+// Output:
+// {}

--- a/interp/gta.go
+++ b/interp/gta.go
@@ -165,7 +165,7 @@ func (interp *Interpreter) gta(root *node, rpath string) ([]*node, error) {
 				return false
 			}
 			if n.child[1].kind == identExpr {
-				n.typ = &itype{cat: aliasT, val: typ, name: typeName, path: rpath}
+				n.typ = &itype{cat: aliasT, val: typ, name: typeName, path: rpath, incomplete: typ.incomplete}
 			} else {
 				n.typ = typ
 				n.typ.name = typeName

